src/node_options.cc

```diff
@@ -179,6 +179,9 @@ void EnvironmentOptions::CheckOptions(std::vector<std::string>* errors,
     } else if (force_repl) {
       errors->push_back("either --watch or --interactive "
                         "can be used, not both");
+    } else if (test_runner_force_exit) {
+      errors->push_back("either --watch or --test-force-exit "
+                        "can be used, not both");
```

Comment:
This is becoming a repetitive check (`if AAA -> push_back("either --watch or AAA can be used, not both")`),
Might be worth extracting to some function 🤔